dnSpy/dnSpy/Documents/Tabs/Settings/AssemblyExplorerAppSettingsPage.cs

```diff
@@ -234,4 +267,14 @@ sealed class MemberKindVM : ViewModelBase {
 			this.Text = text;
 		}
 	}
+
+	sealed class DocumentFilterTypeVM : ViewModelBase {
+		public DocumentFilterType Object { get; }
```

Comment:
Should probably be renamed from Object to FilterType. When I saw the property in code I thought it was a boxed value.